PWGHF/HFJ/CMakeLists.txt

```diff
@@ -7,4 +7,6 @@
 #
 # In applying this license CERN does not waive the privileges and immunities
 # granted to it by virtue of its status as an Intergovernmental Organization
-# or submit itself to any jurisdiction.
\ No newline at end of file
+# or submit itself to any jurisdiction.
+
+add_subdirectory(Tasks)
```

Comment:
I think in the end the new task should go in the PWGJE directory, since it's input are JE tables.